mqtt/mqtt-broker/src/broker.rs

```diff
@@ -88,7 +88,11 @@ where
                         }
                         SystemEvent::AuthorizationUpdate(update) => {
                             if let Err(e) = self.authorizer.update(update) {
-                                warn!(message = "an error occurred updating authorization info", error = %e);
+                                warn!(message = "an error occurred while updating authorization info", error = %e);
```

Comment:
nit: if we are going to shutdown the broker, let's treat this log entry as `error!`